src/mount/mount.ceph.c

```diff
@@ -327,35 +461,29 @@ static int parse_options(const char *data, struct ceph_mount_info *cmi)
 			/* Only legacy ms_mode needs v1 addrs */
 			v2_addrs = strcmp(value, "legacy");
 			skip = false;
+		} else if (strcmp(data, "mon_host") == 0) {
```

Comment:
@jtlayton @batrick @luis-henrix We should have this as `mon_addr` to be in-sync with the option passed to kclient. Thoughts?

Reply:
Yes. I think so, having a mix of subtly-different mount option names sounds like a recipe for confusion.

Reply:
Sure. I'll push an update soon.